packages/fiori-app-sub-generator/src/fiori-app-generator/fioriAppGenerator.ts

```diff
@@ -264,6 +265,14 @@ export class FioriAppGenerator extends Generator {
             }
             // get project information
             if (hasStep(this.fioriSteps, STEP_PROJECT_ATTRIBUTES)) {
+                let promptSettings = generatorOptions.promptSettings?.['@sap/generator-fiori'];
+                if (
+                    this.state.floorplan === FloorplanFE.FE_FPM &&
```

Comment:
Rather than adding more complexity here in the main generator code, can you move the logic into [createUI5ApplicationPromptOptions](https://github.com/SAP/open-ux-tools/blob/635102f462f746f5e4c729bcf701a0c3cb383d7b/packages/fiori-app-sub-generator/src/fiori-app-generator/prompting.ts#L212), thre is some handling their already for ui5 version. Would be good to keep the logic in one place